_data/software/wix-toolset.yml

```diff
@@ -0,0 +1,39 @@
+---
+schemaVersion: '1.0'
+description:
+  whatItDoes:
+    en: >-
+      The WiX toolset lets developers create installers for Windows Installer,
+      the Windows installation engine.
+    fr: >-
+      Les outils WiX permet aux développeurs de créer des programmes
+      d'installation pour Windows Installer, le moteur d'installation de
+      Windows.
+category: '43233400'
+homepageURL:
+  en: 'https://wixtoolset.org/'
+  fr: 'https://wixtoolset.org/'
+licences:
+  - URL:
+      en: 'https://github.com/wixtoolset/Core/blob/master/LICENSE.TXT'
```

Comment:
Not sure if this one is better https://github.com/wixtoolset/wix3/blob/develop/LICENSE.TXT